### ISSUE TITLE
Illegal callback invocation from native module. This callback type only permits a single invocation from native code

### ISSUE THREAD

**Issue by suresh777** [rmcfarlane82/react-native-msal-plugin#36]
I updated to Xcode 11.1 after that i am trying to run msal authentication in simulator. From token async i am getting  this error

Illegal callback invocation from native module. This callback type only permits a single invocation from native code.

RCTFatal
checkCallbackMultipleInvocations(bool*)
__41-[RCTModuleMethod processMethodSignature]_block_invoke_2.129
__123-[RNMsalPlugin authority:clientId:scopes:extraQueryParams:loginHint:msalUIBehavior:extraScopesToConsent:resolver:rejecter:]_block_invoke
__177-[MSALPublicClientApplication acquireTokenForScopes:extraScopesToConsent:account:loginHint:uiBehavior:extraQueryParameters:claims:authority:correlationId:apiId:completionBlock:]_block_invoke
__177-[MSALPublicClientApplication acquireTokenForScopes:extraScopesToConsent:account:loginHint:uiBehavior:extraQueryParameters:claims:authority:correlationId:apiId:completionBlock:]_block_invoke.167
__30-[MSALInteractiveRequest run:]_block_invoke_2
_dispatch_call_block_and_release
_dispatch_client_callout
_dispatch_main_queue_callback_4CF
__CFRUNLOOP_IS_SERVICING_THE_MAIN_DISPATCH_QUEUE__
__CFRunLoopRun
CFRunLoopRunSpecific
GSEventRunModal
UIApplicationMain
main
start
0x0

**Comment by sm1th** [comment]
Is your simulator running iOS 13?

**Comment by suresh777** [comment]
Yes , yesterday i downloaded Xcode 11 and i am trying to run i am getting above exception in Xcode 11 simulators when trying msal login. But . if i run same code in Xcode 10 simulators those are working fine

**Comment by sm1th** [comment]
Yeah it's because Xcode 11 by default only has iOS 13 simulators, and this plugin does not work with iOS 13. You can create an iOS 12 simulator again but you first have to download the iOS 12 runtime. Just go to Xcode > Devices and Simulators > simulators > Add > choose iOS version

**Comment by suresh777** [comment]
I can download iOS 12 simulators but this plugin should support latest changes in iOS and Android right. We are using this plugin for authentication, Today i saw message from contributor that use AppCenter Auth. What should we do
